parts/k8s/manifests/kubernetesmaster-cloud-controller-manager.yaml

```diff
@@ -15,17 +15,24 @@ spec:
       imagePullPolicy: IfNotPresent
       command: ["cloud-controller-manager"]
       args: [<config>]
```

Comment:
could you also update args for ccm according to https://github.com/kubernetes/cloud-provider-azure/blob/master/examples/out-of-tree/cloud-controller-manager.yaml#L16-L28? especially, `"--controllers=*,-cloud-node"` is a new configuration.